packages/deploy-aws/src/templates/seagull_project.ts

```diff
@@ -79,13 +80,17 @@ export class SeagullProject {
     app.stack.addLogGroup(`/aws/lambda/${name}-lambda-handler`)
     app.stack.addLogGroup(`/${name}/data-log`)
     const cronJson = await buildCronJson(this.appPath)
-    cronJson.forEach((rule: Rule) => {
-      app.stack.addEventRule(rule, lambda)
-    })
-
+    cronJson.forEach((rule: Rule) => app.stack.addEventRule(rule, lambda))
+    await this.customizeStack(app)
```

Comment:
I would suggest to put that part to the deployProject/diffProject part, becuase it is clearly an additonal part in the process and should not be mixed up with the initial bootstrap

Reply:
Done!